image/fetcher.go

```diff
@@ -87,7 +86,15 @@ func (f *Fetcher) pullImage(ctx context.Context, imageID string) error {
 		return err
 	}
 	writer := logging.GetInfoWriter(f.logger)
-	termFd, isTerm := term.GetFdInfo(writer)
+	type descriptor interface {
+		Fd() uintptr
+	}
+	var termFd uintptr
+	var isTerm bool
```

Comment:
Minor: Maybe `isTerm := false` for clarity?

Reply:
IMHO it's good practice to use zero values for this kind of thing.